lib/src/executor/host.rs

```diff
@@ -2142,7 +2142,7 @@ impl ReadyToRun {
                     return HostVm::Error {
                         error: Error::Utf8Error {
                             function: host_fn.name(),
-                            param_num: 2,
+                            param_num: 1,
```

Comment:
Very small side fix. Usually I like to open separate PRs, but this is not worth it.